pkg/time/time.go

```diff
@@ -16,10 +16,25 @@ func Open(l *lua.State) {
 
 var timeLibrary = []lua.RegistryFunction{
 	{"now", now},
+	{"nowUTC", nowUTC},
```

Comment:
It would be preferable to have a `time.format` function that takes a number representing nanoseconds since epoch, a layout (described by https://golang.org/pkg/time/#Time.Format), and a location (described by https://golang.org/pkg/time/#LoadLocation).